src/app/actions/SearchActions.js

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
+import { Actions } from './Actions';
+import appConfig from '../../../appConfig';
+
+export const searchResults = (results) => {
+  return {
+    type: Actions.SEARCH,
+    results,
+  };
+};
+
+export const search = (query, filter = 'q') => {
+  // Need a parsed query input to use for each filter
```

Comment:
Are "filters" Elastic filters or a redux component? If they're for Elastic you shouldn't need them to make queries. Also if we intended to use filters we'll probably need to POST a more complex JSON block to Elastic, the URI search doesn't expose all of the Elastic functionality

Reply:
My thinking was that `filters` would be used to match fields in Elastic and I believe should to be captured in a store for keeping track of search history for proper unfiltering and browser history storage for going back via the browser's Back button. I know that tracking history can be tricky, but perhaps Redux can help with this.

We'll definitely need to work on how we're sending queries from the front end to the API and formulating the proper query for sending along to Elastic to take advantage of Elastic's features. Additional work will need to be done on the Search API to parse what's being sent from the front end, send the proper query, and handle the response properly for delivery back to the front end

I'll need to do some research on this, and I know Paul has wrangled with query handling and it can be problematic to get right.

Reply:
Got it, that makes sense. I was a little confused because I see `filters` as more of an advanced search/browse functionality, but of course it should be supported here. I think our work on the API will help inform this in the next iterations, especially since it will provide at least _some_ constraints on how we pass data back to Elastic through the API